xla/service/gpu/gemm_rewriter.cc

```diff
@@ -2261,10 +2268,14 @@ class GemmWorkspaceRewriteVisitor : public DfsHloRewriteVisitor {
       TF_ASSIGN_OR_RETURN(
           has_aux_output,
           xla::gpu::gpublas_lt::EpilogueHasAuxiliaryOutput(epilogue));
+      if (config.has_damax_output()) {
+        has_damax_output = config.damax_output();
```

Comment:
No need to check if the config has damax_output, you can just set `has_damax_output` uncondtionally. 

I don't think you even need a `has_damax_output` local variable. You can just use `config.damax_output()` in the if-statement below